.gitignore

```diff
@@ -24,4 +23,4 @@ docker-compose.yml
 /golbat
 /geojson/*
 vbase.proto
-__debug_bin
\ No newline at end of file
+__debug_bin
```

Comment:
all that's going on here is that vim just added the missing newline to the end of file here.